app/models/request.rb

```diff
@@ -11,7 +11,7 @@ class Request < ApplicationRecord
   scope :with_roster, ->(roster_id) { joins(:roster, :user).where(roster_id:).includes(:roster, :user, :offers, roster: [:patrol]) }
   scope :with_requested_by, ->(user_id) { where(user_id:) }
   scope :with_successful_status, -> { where(status: 'successful') }
-  scope :with_open_status, -> { where(status: 'open') }
+  scope :with_open_status, -> { joins(:roster).where(status: 'open').where('finish > ?', DateTime.now) }
```

Comment:
Looks good! Some specs would be nice here, especially if it came from a bug. Happy to flesh something out on this PR if you want?

Reply:
Thank you for offering. I would love help on writing specs... once I know how I can do myself

Reply:
On reflection, there are currently no specs on any scopes, and I'd like to add more than just this one. 
Let's tackle specs in a separate PR. I'll smerge this once the build passes.